wger/settings_global.py

```diff
@@ -380,3 +396,30 @@
     'EMAIL_FROM': 'wger Workout Manager <wger@example.com>',
     'TWITTER': False
 }
+
+SOCIAL_AUTH_FACEBOOK_KEY = ''
```

Comment:
Set these to get the values from the environment variables

Reply:
I have updated the file